docs/_reference/use-cases/ai-app.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Zapier integration structure for an AI app
+order: 4
+layout: post-toc
+redirect_from: 
+
+---
+
+# Zapier integration structure for an AI app
+
+AI app integrations built on Zapier allow users to automate tasks using AI capabilities. Here are some common pain points and recommendations when building AI apps on Zapier.
+
+## Pain Points
+### 1. Long-Running Actions  
+
+**Pain Point:** Actions that take a long time to process can cause issues, especially with Zapier's execution time limits. Test steps in the Zapier editor are limited to 50 seconds, and live Zap executions default to 45 seconds.
+
+**Recommendations:**
+
+* Use `z.generateCallbackUrl()` and `performResume`
```

Comment:
add link to https://github.com/zapier/zapier-platform/blob/main/packages/cli/README.md#zgeneratecallbackurl